### PR TITLE
change style & color scheme of page

### PR DESCRIPTION
**Pull request by haoyxc**
Change the layout of page and add colors. Mostly just wanted to put this here to get some concrete feedback!!
LATEST UPDATE:
image

Hover over a button:
Screenshot 2020-07-29 at 4 34 49 PM

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -10,38 +10,43 @@
...
-      <label>Zoom on Node in Graph: </label>
-      <input type="text" id="node-search" placeholder="Node Name"></input>
-      <button id="search-button">Find Node</button>
-      <label id="node-error" class="error-text"></label>
+  </h2>
+  <div id="input-boxes">
+    <!-- Box for filtering the graph -->
+    <div id="search-box" class="input-child">
+      <h4 class="search-title">Filter the Graph</h4>
+      <label for="layers">Radius from filtered node(s) (0-20, default is 3):</label>
```
Comment (IshaniSanturkar) [inline src/main/webapp/index.html:20]: Do you think we should rethink this 20 thing now?

File: src/main/webapp/style.scss [hunk 2]
```diff
@@ -4,16 +4,47 @@
...
 }
 
+.search-title {
+  margin-bottom: 20px;
+}
+
+button {
+  background-color: var(--gray);
+  color: black;
+    font-size: 16px;
```
Comment (IshaniSanturkar) [inline src/main/webapp/style.scss:26]: Fix indentation here.

Comment (IshaniSanturkar) [reply src/main/webapp/style.scss:26]: Also `outline: none` removes the weird black outline we see when the button is clicked.

File: src/main/webapp/index.html (hunk 1 above)
Comment (haoyxc) [reply src/main/webapp/index.html:20]: what do you think it should be?

Comment (IshaniSanturkar) [reply src/main/webapp/index.html:20]: Maybe the actual max depth? But then we would have to add a backend parameter.

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -10,38 +10,43 @@
...
-    <button id="reset" class="reset">Reset Graph</button>
-    <label id="num-selected"></label>
-    <br />
-    <div id="num-mutation-display">
-      <p id="graph-number-text">Graph </p>
-      <input type="number" id="graph-number" min="0" max="0" value="0"
-        onchange="graph.readGraphNumberInput(); graph.generateGraph()">
-      <p id="total-mutation-number-text">out of 0</p>
+        <label>Zoom on Token in Graph: </label>
+        <input type="text" id="token-search" placeholder="Token Name"></input>
```
Comment (IshaniSanturkar) [inline src/main/webapp/index.html:44]: This is kind of a general question, but what is the relationship between the two zooms? We can't do them both at the same time right?

File: src/main/webapp/style.scss [hunk 4]
```diff
@@ -4,30 +4,67 @@
...
+
+button {
+  background-color: var(--gray);
+  color: black;
+  font-size: 16px;
+  border-radius: 10px;
+  margin: 5px;
+  padding: 5px;
+  border-style: solid;
+  border-width: 0;
```
Comment (IshaniSanturkar) [inline src/main/webapp/style.scss:35]: Just curious, what is the purpose of border styling if the width is 0 :)

**Pull request merged**